src/agent/inspectordebugapi.ts

```diff
@@ -33,9 +33,9 @@ import {V8Inspector} from './v8inspector';
 
```

Comment:
Not part of this change, so this can be fixed in a follow-on. It would be good to use snake-case for file names.